assume/world.py

```diff
@@ -89,7 +89,7 @@ def __init__(
         addr: Union[tuple[str, int], str] = "world",
         database_uri: str = "",
         export_csv_path: str = "",
-        log_level: str = "INFO",
+        log_level: str = "WARNING",
```

Comment:
why do we want to log only warning? seeing the progress of loading and others is pretty helpful. Or is this for a special case only?